cpp/oneapi/dal/table/backend/interop/host_homogen_table_adapter.hpp

```diff
@@ -17,6 +17,7 @@
 #pragma once
 
 #include <daal/include/data_management/data/homogen_numeric_table.h>
+#include <daal/include/data_management/data/soa_numeric_table.h>
```

Comment:
Are declarations from these files used in this header? If not, include them in `.cpp`.

Reply:
moved to .cpp